src/canvas/PatternPath.tsx

```diff
@@ -1,8 +1,12 @@
 import { Point } from './Point';
 import { PatternPathType } from './Enums';
 
+const MIN_SQUARED_DISTANCE_BETWEEN_POINTS = 25;
```

Comment:
I think these constants could be better scoped. Since they are only used but one function, move them to the function they are used in.

Reply:
ok